tensorflow/core/kernels/linalg/matrix_solve_op.cc

```diff
@@ -114,9 +118,11 @@ class MatrixSolveOp : public LinearAlgebraOp<Scalar> {
   TF_DISALLOW_COPY_AND_ASSIGN(MatrixSolveOp);
 };
 
-#if GOOGLE_CUDA
+#if GOOGLE_CUDA || TESNORFLOW_USE_ROCM
```

Comment:
typo --- `TESNORFLOW_USE_ROCM`  !!

that also implies the changes contained within are not getting picked up :(